lib/session-middleware.js

```diff
@@ -1,49 +1,45 @@
-import cookie from 'cookie';
-import cookieParser from 'cookie-parser';
+import cookie from 'cookie'
+import cookieParser from 'cookie-parser'
 
 /** PRIVATE **/
-function setInvalidSession(socket) {
-  socket.session = {};
-  socket.session.save = function (callback) {
-    callback = callback || function () { };
-    callback('Session is not initialized. Init it with a classic express request.');
-  };
+const setInvalidSession = (socket) => {
```

Comment:
With the arrow functions you can drop the `()` with a single param.
`= socket =>` vs `= (socket) =>`

Reply:
Oh, I did not know this shortcut! But we should keep those parenthesis for clarity ;)